<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PYQTProjectManager</class>
 <widget class="QMainWindow" name="PYQTProjectManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PYQTProjectManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createProjectBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать новый проект</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройики</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>89</y>
      <width>511</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
